docs/contributer_guide/install.rst

```diff
@@ -0,0 +1,56 @@
+Developer Installation
+======================
+
+For using ``ngshare``, see :doc:`../user_guide/install`.
+
+Install from GitHub
+----------------
+.. code:: bash
+
+    git clone https://github.com/lxylxy123456/ngshare.git
+    cd ngshare/
+    pip3 install .
+
+Run Installed ngshare
+---------------------
+.. code:: bash
+
+    python3 -m ngshare [arguments]
+
+Run ngshare without Installation
+--------------------------------
+The first line installs pip dependencies
+
+.. code:: bash
+
+    pip3 install tornado jupyterhub sqlalchemy
+    git clone https://github.com/lxylxy123456/ngshare.git
+    cd ngshare/ngshare/
+    python3 ngshare.py [arguments]
+
+Run vngshare
+------------
+vngshare is simply adding some arguments to ngshare.
+
+* ``--vngshare``: Mock authentication (using only username)
+* ``--debug``: enable debug
+* ``--database sqlite:////tmp/ngshare.db``: change default database path
+* ``--storage /tmp/ngshare``: change default storage path
```

Comment:
It's hard to understand what this part is saying. My guess is that the first sentence means "vngshare is a part of ngshare and can be enabled with the following command line arguments." If this is the case, why would someone need to specify options other than ``--vngshare``?

Reply:
This is re-worded to: "vnshare can be used by running ``vngshare.py`` or by adding some arguments to ngshare."

Reply:
Are all options required to enable vngshare, or is it just `--vngshare`?

Reply:
Well, there is no definition for vngshare. It is just ngshare with some alternation. 
`--vngshare` changes authentication, `--debug` is for debug (e.g. clear database). The others are setting default db location (or it will access `/srv` and get permission denied). 

Considering the description of vngshare in the docs, I think `--vngshare -- debug` are required.

Reply:
OK.
I think some readers may be confused why this section shows the database and storage options but the sections below don't. I was thinking about removing the unrequired options, but it seems to be complicated.